mantidproject/mantid
# Document and Rename Damping Minimizer to Damped Gauss-Newton

This is part of the documentation and improvement of fitting minimizers. 
This will be mentioned in the release notes later on.

**To test:**
Build HTML documentation.
Check the documentation to find the documentation for DampedGaussNewtonMinimizer exists, 
the Damping minimizer has been renamed to this and 
other occurrences of "Damping"  have not been accidentally changed.

Check all documents that may have been modified by these changes.

Also build QTHelp and check the Mantid help.

Check that the DampedGaussNewtonMinimizer can be selected and perhaps do a fit with it.


Fixes #18003.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
